common/examples.js

```diff
@@ -224,6 +249,10 @@ $(document).ready(function() {
 			applyByFunctionFilter(event.target.value)
 		});
 	})
+    .fail(function(jqXHR, textStatus, error) {
+        const err = textStatus + ". " + error;
+        console.error("Error loading function_usages.json: " + err);
+    });
```

Comment:
i added this error handler to both. in case there is some issue it will no longer silently fail and whoever is working can fix the problems